src/game.zig

```diff
@@ -941,6 +947,8 @@ pub fn update(deltaTime: f64) void { // MARK: update()
 		const fricMul = speedMultiplier*baseFrictionCoefficient*if(Player.isFlying.load(.monotonic)) 1.0 else mobility;
 
 		const forward = vec.rotateZ(Vec3d{0, 1, 0}, -camera.rotation[2]);
+		const lerpAmount: Vec3d = @splat(std.math.clamp(density/@max(1.0, maxDensity), 0.0, 1.0));
+		const lerpedDir = if(density > 0.1) std.math.lerp(forward, camera.direction, lerpAmount) else forward;
```

Comment:
Do we really need the `if(density > 0.1)` to me it looks like you are trying to fix a bug that doesn't exist, since at air density both of these should be almost equal.